packages/stores/src/account/__tests_e2e__/positions.spec.ts

```diff
@@ -206,6 +205,82 @@ describe("Create CL Positions Txs", () => {
     ).rejects.toBeDefined();
   });
 
+  it("can have liquidity be partially removed from it", async () => {
+    await createFullRangePosition(queryPool!.id);
+
+    const userPositionIds = await getUserPositionsIdsForPool(queryPool!.id);
+    const lastPositionId = userPositionIds[userPositionIds.length - 1];
+
+    // get query position
+    const position = queriesStore
+      .get(chainId)
+      .osmosis!.queryAccountsPositions.get(account.bech32Address)
+      .positions.find(({ id }) => id === lastPositionId);
+    await position?.waitFreshResponse();
+
+    if (!position || !position.liquidity) throw new Error("Position not found");
+
+    const tx: any = await new Promise((resolve, reject) =>
+      account.osmosis.sendWithdrawConcentratedLiquidityPositionMsg(
+        lastPositionId,
+        position.liquidity!.mul(new Dec(0.5)),
+        undefined,
+        (tx) => {
+          if (tx.code) reject(tx.log);
+          else resolve(tx);
+        }
+      )
+    );
+
+    expect(tx).toBeDefined();
+
+    // old position is still there
+    const userPositionIds_after = await getUserPositionsIdsForPool(
+      queryPool!.id
+    );
+    const lastPositionId_after =
+      userPositionIds_after[userPositionIds_after.length - 1];
+    expect(lastPositionId_after).toBe(lastPositionId);
+  });
+
+  it("can have liquidity be entirely removed from it (deletes position)", async () => {
+    await createFullRangePosition(queryPool!.id);
+
+    const userPositionIds = await getUserPositionsIdsForPool(queryPool!.id);
+    const lastPositionId = userPositionIds[userPositionIds.length - 1];
+
+    // get query position
+    const position = queriesStore
+      .get(chainId)
+      .osmosis!.queryAccountsPositions.get(account.bech32Address)
+      .positions.find(({ id }) => id === lastPositionId);
+    await position?.waitFreshResponse();
+
+    if (!position || !position.liquidity) throw new Error("Position not found");
+
+    const tx: any = await new Promise((resolve, reject) =>
```

Comment:
How about we use `unknown` instead of `any` here? `unknown` is like a type-safe version of `any`.